.github/CODEOWNERS

```diff
@@ -1,3 +1,4 @@
 # This is the module where the Android Design System resides
 # Any changes to the source files of this module require approval from @malmstein or @nalcalag.
-/common-ui/ @malmstein @nalcalag
\ No newline at end of file
+/common/common-ui/ @malmstein @nalcalag
+/saved-sites/ @malmstein
```

Comment:
what's the configuration for codeowners, to ping named people as an FYI or to require a review from them?

Reply:
That’s correct @CDRussell.